src/client/visualizations/table/table.tsx

```diff
@@ -452,8 +464,17 @@ export class Table extends BaseVisualization<TableState> {
       <div className="highlight">{highlighter}</div>
     </div>;
 
+    const defaultSegmentWidth = this.defaultSegmentWidth();
     const corner = <div className="corner">
       <div className="corner-wrap">{segmentTitle}</div>
+      <ResizeHandle
+        direction={Direction.LEFT}
+        onResize={this.setSegmentWidth}
+        min={defaultSegmentWidth}
+        max={MAX_SEGMENT_WIDTH}
+        initialValue={segmentWidth || defaultSegmentWidth}
+        children={null}
```

Comment:
You pass null and check for undefined. Scary ;)

Reply:
Yeah 😛 As the only use case is to not render the icon, I change it to a flag to skip it.